.github/workflows/test.yaml

```diff
@@ -80,7 +79,7 @@ jobs:
           --show-error
           --request DELETE
           --header "Accept: application/vnd.github.v3+json"
-          --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}"
+          --header "Authorization: token ${{ github.token }}"
```

Comment:
Commit message: it's typically assumed that relative filepaths are relative to the root of the repository, so I would either say `.github/workflows/test.yaml` or simply `test.yaml` in this case, since no other file in this repository bears that name.